static_report/src/components/ComparisonReport/CRTableColumnDetails.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Code, Flex, Text } from '@chakra-ui/react';
+import { formatNumber, getColumnDetails, getMissingValue } from '../../utils';
+import { MetricsInfo } from '../shared/MetrisInfo';
+import { ComparisonReportSchema } from '../../sdlc/comparison-report-schema';
+
+// Temp Typing
+type CRTableColumnDetailsProps = {
+  column: any;
+  baseColumn: ComparisonReportSchema['base']['tables']['ACTION']['columns']['DATE'];
+  inputColumn: ComparisonReportSchema['input']['tables']['ACTION']['columns']['DATE'];
+};
+export const CRTableColumnDetails: React.FC<CRTableColumnDetailsProps> = ({
```

Comment:
What do you think about this? It's more readable to me, I thought generic is unnecessary.
```suggestion
export function CRTableColumnDetails(props: CRTableColumnDetailsProps) {}
```

Reply:
I know it seems like boilerplate, but then we don't catch scenarios where we don't actually return JSX type, which we are in this situation
![image](https://user-images.githubusercontent.com/11186480/176800719-a4bce1fb-5114-43f9-9007-4de26ec07818.png)
![image](https://user-images.githubusercontent.com/11186480/176800877-959b7413-8806-4ff3-98f5-9a1745d993cc.png)